src/domain/signupGroup/utils.ts

```diff
@@ -363,3 +368,31 @@ export const calculateTotalPrice = (
       (priceGroupOptions.find((o) => o.value === curr.priceGroup)?.price ?? 0),
     0
   );
+
+export const getSignupPriceGroupOptions = (
+  registration: Registration,
+  locale: Language
+) => {
+  return (
+    registration.registration_price_groups?.map((pg) => {
+      const price = pg?.price ? Number(pg.price) : 0;
+
+      return {
+        label: [
+          `${getLocalisedString(pg?.price_group?.description, locale)}`,
+          `${price.toFixed(2).replace('.', ',')} €`,
+        ].join(' '),
+        price,
+        value: pg?.id?.toString() ?? /* istanbul ignore next */ '',
+      };
+    }) ?? /* istanbul ignore next */ []
+  );
+};
+
+export const shouldCreatePayment = (
+  priceGroupOptions: SignupPriceGroupOption[],
+  signups: SignupFormFields[]
+) =>
+  featureFlagUtils.isFeatureEnabled('WEB_STORE_INTEGRATION') &&
+  calculateTotalPrice(priceGroupOptions, signups) > 0 &&
+  Boolean(signups.find((su) => !su.inWaitingList));
```

Comment:
The back-end expects all signups to be in the "attending" status before a payment can be made (when creating a payment for a group). If my slowly deteriorating JS understanding is still valid, this function would allow to create a payment if only one signup in a group is in the attending status, but the rest are not.

Reply:
This is very good catch. In theory it should not possible to have signups with different attendee status values, but current implementation of `shouldCreatePayment` function could cause hard to find bugs if that logic changes at some point.

Changes logic so that all signups have to be attendees in the event to create a payment.